internal/workload/podman/podman.go

```diff
@@ -515,8 +515,6 @@ func (p *podman) containerLog(ctx context.Context, containerID string, res io.Wr
 					log.Errorf("cannot write log line to io.Writer for container '%v': %v", containerID, err)
 				}
 			case <-ctx.Done():
-				close(stderrCh)
```

Comment:
I think we should close the producer goroutine also. If you close only this one, data keeps accumulating into the channel for nothing.

Reply:
data into the channel is not a problem. but if no one closes the routine then we may have a leak. I will check it.

Reply:
@tupyy I opened a separate bug for this issue: https://issues.redhat.com/browse/ECOPROJECT-842

Reply:
@jakub-dzon yes